backend/src/test/java/codezap/template/service/facade/TemplateApplicationServiceTest.java

```diff
@@ -202,23 +202,35 @@ void findAllBy(
             // given
             var member = memberRepository.save(MemberFixture.getFirstMember());
             var category = categoryRepository.save(Category.createDefaultCategory(member));
-            var template1 = templateRepository.save(new Template(member, "title1", "description", category));
-            var sourceCode1 = sourceCodeRepository.save(new SourceCode(template1, "filename1", "content", 1));
-            thumbnailRepository.save(new Thumbnail(template1, sourceCode1));
+
+            var template = templateRepository.save(new Template(member, "title1", "description", category));
+            var sourceCode = sourceCodeRepository.save(new SourceCode(template, "filename1", "content", 1));
+            thumbnailRepository.save(new Thumbnail(template, sourceCode));
```

Comment:
⚡️ 이거 추가된 3개 메서드 모두에서 사용 중인 것 같은데, 메서드로 빼면 어떨까요??

그리구 픽스처가 있는 친구들은 재활용해볼까요?~ 의미없는 상수로부터 눈이 조금이나마 자유로워질 수 있을 것 같아요
``
```suggestion
            var template = templateRepository.save(TemplateFixture.get(member, category));
            var sourceCode = sourceCodeRepository.save(SourceCodeFixture.get(template, 1));
            thumbnailRepository.save(new Thumbnail(template, sourceCode));
```